docs/notebooks/example1-with-sensors.xml

```diff
@@ -223,6 +223,27 @@
                     <LinkedTimeSeriesID IDref="TS-ResourceUse-Electricity-Energy-11"/>
                     <LinkedTimeSeriesID IDref="TS-ResourceUse-Electricity-Energy-12"/>
                   </AnnualFuelUseLinkedTimeSeriesIDs>
+                  <Emissions>
+                    <Emission>
+                      <EmissionBoundary>Indirect</EmissionBoundary>
+                      <EmissionsType>CO2e</EmissionsType>
+                      <GHGEmissions>25000</GHGEmissions>
+                      <EmissionsLinkedTimeSeriesIDs>
+                        <LinkedTimeSeriesID IDref="TS-ResourceUse-CO2e-1"/>
```

Comment:
```suggestion
                        <LinkedTimeSeriesID IDref="TS-ResourceUse-Greenhouse Gas Emissions-1"/>
```

Reply:
This needs to be changed from CO2 to Greenhouse Gas Emissions throughout, based on the new element